src/Components/3DV/SceneView.stories.tsx

```diff
@@ -16,6 +35,231 @@ export default {
     }
 };
 
+const defaultGizmoElementItems: TransformedElementItem = {
+    meshIds: ['tank6_LOD0.003_primitive0', 'tank6_LOD0.003_primitive1'],
+    parentMeshId: 'tank6_LOD0.003_primitive0'
+};
+
+export const Gizmo = () => {
+    const [state, dispatch] = useReducer(
```

Comment:
Hmm, maybe we can move all the logic and the returned dom here to a separate standalone wrapper component called like `SceneViewWithGizmo` or something? Having lots of things happening in a story may not be ideal. What do you think?

Reply:
Yeah that might be good -- would I just generate a new component (complete with types, stories, etc.) or just move it all the a standalone .tsx file?